copy/copy.go

```diff
@@ -100,6 +102,7 @@ type Options struct {
 	Progress         chan types.ProgressProperties // Reported to when ProgressInterval has arrived for a single artifact+offset.
 	// manifest MIME type of image set by user. "" is default and means use the autodetection to the the manifest MIME type
 	ForceManifestMIMEType string
+	MultipleImages        bool // If set, expect to copy multiple images
```

Comment:
I keep going back and forth on whether this should be an enumerated value, to be able to request different behavior (for example, "converting" a single image into a multiple image on copy) in the future.

Reply:
Makes sense to me.  Changed.